assets/css/less/admin-notice-review.less

```diff
@@ -0,0 +1,112 @@
+.wpbdp-review-notice {
+	&.notice {
+		border: 1px solid #ddd;
+		text-align: center;
+		position: fixed !important;
+		z-index: 999;
+		bottom: 0;
+		right: 0;
+		width: 400px;
+		margin-bottom: 0;
+		padding-bottom: 15px;
+
+		form, p {
+			font-size: 14px;
+			max-width: 600px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+	}
+	a {
+		margin: 5px;
+	}
+	.wpbdp-button {
```

Comment:
We'll be using these button styles for the whole admin area, so let's include them in a different file.the wait spinner will also be used admin-wide so we should move that as well. Anything else?

Mentioned here as well:
https://github.com/Strategy11/business-directory-plugin/pull/44#discussion_r727229907

Reply:
I will move buttons to their own less file, fore easier management to allow use in the admin area